Felienne/hedy
# replace "FireFox" with "Firefox"

**Description**

Replaces "FireFox" with "Firefox".

**Fix for**

(no issue filed)

**How to test**

- Check the spelling on the [front page](https://www.hedycode.com) (section "Do I need to install anything?" near the bottom of the page) for all languages
- Check the spelling on the ["For teachers" page](https://www.hedycode.com/for-teachers) (requires being logged in to a teacher account) page for all languages (tab "Introduction", section "Devices" near the bottom of said tab)

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
